lib/Core/Memory.cpp

```diff
@@ -229,6 +229,20 @@ const UpdateList &ObjectState::getUpdates() const {
   return updates;
 }
 
+void ObjectState::flushToConcreteStore(TimingSolver *solver,
+                                       const ExecutionState &state) const {
+  for (unsigned i = 0; i < size; i++) {
+    if (isByteKnownSymbolic(i)) {
+      ref<ConstantExpr> ce;
+      bool success = solver->getValue(state, read8(i), ce);
+      if(!success) 
+        klee_warning("Solver timed out when getting a value for external call, byte %u will have random value", i);
```

Comment:
I don't think the byte number is helpful info, perhaps use the actual address or add the base address?  But not a big deal.

Reply:
I think offset is more useful than the pointer. For example if you print, you will see something weird in Nth position, whereas a pointer is meaningless in this context.

Maybe it's just better to not outout anything?

Reply:
Either nothing, or add the base address too.  I feel just an offset w/o any context is unusual.

Reply:
@kren1 I second @ccadar's comment. The base pointer (plus index) often helps in debugging.